README.md

```diff
@@ -16,9 +16,16 @@ Also called the Empower Plant UI/UX. This project was bootstrapped with [Create
 | sentry-logback | 5.5.1 |
 
 ## Setup
-1. Permit your IP address in CloudSQL.
+1. Permit your IP address in CloudSQL by posting a message in the `#se-team` Slack channel.
 2. Copy `local.env` from [application-monitoring-config](https://github.com/sentry-demos/application-monitoring-config) into `env-config` directory of your local repo, or, if you don't have access to it, follow `env-config/example.env`.
-4. The `REACT_APP_FLASK_BACKEND` in `env-config/*.env` points to the backend instance deployed to AppEngine, the same one used by the cloud-hosted React web app. Flask is the default backend. If you expect to run other backend types, add values for those in `env-config/*.env` as well (i.e. `REACT_APP_EXPRESS_BACKEND`).
+3. The `REACT_APP_FLASK_BACKEND` in `env-config/local.env` points to the backend instance deployed to AppEngine, the same one used by the cloud-hosted React web app. Flask is the default backend. If you expect to run other backend types, add values for those in `env-config` in your `local.env` file as well (i.e. `REACT_APP_EXPRESS_BACKEND`).
+4. Confirm [Homebrew](https://brew.sh/) is installed with `brew -v`. If not, install using `/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"`.
+5. Confirm PostgreSQL is installed with `Postgres -V`. If not, install using `brew install postgresql`.
+6. As the application is compatible with specific versions of `node` and `npm`, install the following to be able to set the specific versions required:
+    1. Install compatible `npm` version with `npm install -g npm@XX.XX.XX`. NOTE: may need to use `sudo` with command.
+    2. Install `n` to update `node` version with `npm install -g n`.
+    3. Set the specific `node` version with `n XX.XX.XX`. NOTE: may need to use `sudo` with command.
```

Comment:
cool, I didn't know about `n`
I used `nvm` to do that. Is `n` preferable?

Reply:
I've used both! Let me see which one is more preferred. Looking into now...

Reply:
Here are the main differences between `n` and `nvm`. I'm open to changing this to `nvm` since it is more widely known. Let me know what you think is best!
```
The main advantage of n's approach to managing Node.js releases compared to NVM is that global npm packages are not affected when you switch between different versions of Node. On the other hand, NVM lets you use different Node.js versions in different terminals through its nvm use command, while n does not. Once you switch to a different Node.js version, the change is reflected system wide. You can get around this by using the n use subcommand to invoke a specific Node.js binary without switching to that version. This can come in handy when performing one-off script executions.
```
https://www.honeybadger.io/blog/node-environment-managers/

Reply:
Thanks for the summary, that's good to know. Let's leave `n` since you already tested those commands. I'll add a "alternatively you can use nvm" some other time.